features/step_definitions/task_steps.rb

```diff
@@ -10,7 +10,16 @@
 
 Given /the following Iteration exist:$/ do |table|
   table.hashes.each do |hash|
-    Iteration.create hash
+    iter = Iteration.create hash
+    iter.set_timestamp("2012/2/4", "2022/2/4")
```

Comment:
?

Reply:
The iteration starting date and end date has to be set. So I add the following.